Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Helpers/SmartWeakEvent.cs

```diff
@@ -0,0 +1,174 @@
+﻿// ******************************************************************
```

Comment:
I would say: go with regular events. We need to stick with what is already in the toolkit and in the SDK.

If people want weak event they can use the WeakEventListener :)